config/secrets.yml

```diff
@@ -14,7 +14,7 @@ development:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
 test:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 7053b205c9559f83960f18d23f10d36cacd737c0e17e0dd71cbb90d2f3e6
```

Comment:
@mfojtik `SECRET_KEY_BASE` is not part of the build env, only the pod template.

It should not be part of the build, as we don't want that *secret* baked in the image, right?

Do you have any concern with having `test.secret_key_base` stored in the repository?